<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>160</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"> QLabel {
    qproperty-alignment: AlignCenter;
    padding: 5px 0px;
    color: rgb(0, 0, 0);
    max-height: 18px;
    font-size: 16px;
}</string>
     </property>
     <property name="text">
      <string>Go to a HKL</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="1" column="2">
      <widget class="QLineEdit" name="L_set">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>38</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>75</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>H</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_3">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>75</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>L</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="H_set">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>38</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_2">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>75</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>K</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="K_set">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>38</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="calc_HKL">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>38</height>
      </size>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>239</red>
           <green>235</green>
           <blue>231</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>239</red>
           <green>235</green>
           <blue>231</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>239</red>
           <green>235</green>
           <blue>231</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>GO</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
